src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -191,7 +191,7 @@ const AccessibilityRequestDetailPage = () => {
     </>
   );
 
-  const bodyNoDocuments = (
+  const bodyNoDocumentsBasicUser = (
```

Comment:
I don't love the name `BasicUser`, but I do understand why it's that way: user without job codes. I think this is shown to a "business owner". I think reflecting that this is shown to a business owner rather than a basic user might be more clear.

Not a major/blocking comment.

Reply:
Makes sense.  I'm still learning the terminology for this project. . .changed it to `bodyNoDocumentBusinessOwner`.